tests/foreman/ui/test_syncplan.py

```diff
@@ -119,7 +90,6 @@ def test_positive_create_with_start_time(self):
             )
             self.assertIsNotNone(self.syncplan.search(plan_name))
             self.syncplan.search(plan_name).click()
-            self.syncplan.wait_for_ajax()
```

Comment:
People put that for some reason here. It can be a reason for failure on Jenkins or something.
That `element.click()` is not the same as `self.click()`, so we do not have `wait_for_ajax()` by default for it.